dashboard/test/controllers/activities_controller_test.rb

```diff
@@ -1343,4 +1341,15 @@ def new_level
     post :milestone, params: milestone_params
     assert_response 403
   end
+
+  test "milestone strips emoji from program and saves it" do
+    params = @milestone_params
+    params[:program] = panda_panda
+
+    post :milestone, params: params
+
+    user_level = UserLevel.last
+    # panda_panda contains a panda emoji, ensure that it's gone
+    assert_equal user_level.level_source.data, 'Panda'
```

Comment:
The `<hey>` tags are in the previous test (from the `merge` in the `post` request), not this test.